doc/python3.rst

```diff
@@ -20,7 +20,7 @@ with subtype 0.
 For example, let's insert a :class:`bytes` instance using Python 3 then
 read it back. Notice the byte string is decoded back to :class:`bytes`::
 
-  Python 3.6.1 (v3.6.1:69c0db5050, Mar 21 2017, 01:21:04)
+  Python 3.6.2 (v3.6.1:69c0db5050, Mar 21 2017, 01:21:04)
```

Comment:
Here and below, 3.6.1 is still present. Let's use some real output:
```
Python 3.6.8 (v3.6.8:3c6b436a57, Dec 24 2018, 02:04:31)
[GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.57)] on darwin
```

Reply:
Done